src/features/package/FieldHelpTemplate.jsx

```diff
@@ -0,0 +1,27 @@
+import { helpId } from "@rjsf/utils";
+import Markdown from "marked-react";
+import React from "react";
+import PropTypes from "prop-types";
+
+export function FieldHelpTemplate({
```

Comment:
Please use `export default` since we are not going to export anything else from here and to be consistent with the other components.